src/pynxtools_raman_multiformat/config/config_file_rod.json

```diff
@@ -0,0 +1,44 @@
+{

```

Comment:
Are these supposed to be default dicts for the different readers. If so, you could do inside the `handle_rod_file` method:
```python
reader_dir = Path(__file__).parent
self.config_file: reader_dir.joinpath("config", "config_file_rod.json")
```

Reply:
yes, they are. At least for the rod files, if implemented this as you advised.